sections/collage.liquid

```diff
@@ -6,296 +6,250 @@
-                        {%- if featured_media.width >= 3000 -%}{{ featured_media | img_url: '3000x' }} 3000w,{%- endif -%}
-                        {{ featured_media | img_url: 'master' }} {{ featured_media.width }}w"
-                      src="{{ featured_media | img_url: '1500x' }}"
-                      sizes="(min-width: {{ settings.page_width }}px) {% if section.blocks.size == 1 %}calc({{ settings.page_width }}px - 100px){% else %}{{ settings.page_width | minus: 100 | times: 0.67 | round }}px{% endif %}, (min-width: 750px){% if section.blocks.size == 1 %} calc(100vw - 100px){% else %} 500px{% endif %}, calc(100vw - 30px)"
-                      alt="{{ featured_media.alt | escape }}"
-                      loading="lazy"
-                      width="{{ featured_media.width }}"
-                      height="{{ featured_media.height }}"
-                      class="collage-card__image"
+          {%- when 'product'-%}
+            <div class="collage-card collage-product card-wrapper{% if focus_card_left %} collage-card--left{% elsif focus_card_right %} collage-card--right{% endif %}">
+              {%- if block.settings.product != blank -%}
+                {%- assign featured_media = block.settings.product.selected_or_first_available_variant.featured_media | default: block.settings.product.featured_media -%}
+                <div class="collage-content card{% if block.settings.secondary_background %} card-colored color-background-1{% endif %}{% if block.settings.image_padding %} collage-card-spacing{% endif %}" {{ block.shopify_attributes }}>
+                  {%- if featured_media != blank -%}
+                    <div class="collage-content__info card-information card-information__wrapper">
+                      <h3 class="card-information__text h4"><a href="{{ block.settings.product.url }}" class="full-unstyled-link">{{ block.settings.product.title | escape }}</a></h3>
+                      <span class="price">{% render 'price', product: block.settings.product %}</span>
+                    </div>
+                    
+                    <div class="collage-card__image-wrapper media media--transparent media--hover-effect"
+                      {% if focus_card_left and forloop.first %}
+                        style="padding-bottom: {{ 1 | divided_by: featured_media.aspect_ratio | times: 100 }}%;"
+                      {% elsif focus_card_right and forloop.last %}
+                        style="padding-bottom: {{ 1 | divided_by: featured_media.aspect_ratio | times: 100 }}%;"
+                      {% endif %}
                     >
-
-                    {%- if block.settings.second_image and block.settings.product.media[1] != nil -%}
                       <img
-                        srcset="{%- if block.settings.product.media[1].width >= 550 -%}{{ block.settings.product.media[1] | img_url: '550x' }} 550w,{%- endif -%}
```

Comment:
It shouldn't be, I think the the diffs just look weird because I added a div that encompasses all this. You'll notice the line being compared to this is a different part of the file (line 68 vs 87).